kfs-core/src/main/java/org/kuali/kfs/vnd/document/validation/impl/VendorRule.java

```diff
@@ -82,6 +84,9 @@
 
     private VendorDetail oldVendor;
     private VendorDetail newVendor;
+    
+    private static final String VENDOR_SUPPLIER_DIVERSITY_CODE = "vendorSupplierDiversityCode";
```

Comment:
Please avoid "magic strings". Constants (from a constants file) should be used instead.